src/containers/DefaultLayout.js

```diff
@@ -83,107 +87,102 @@ class DefaultLayout extends React.Component {
-              theme="dark"
-              mode="inline"
-              onClick={this.handleNav}
-              selectedKeys={[this.state.currentTab]}
-            >
-              <Menu.Item key="map">Map</Menu.Item>
-              <Menu.Item key="networks">Table View</Menu.Item>
-              <Menu.Item key="resources">Guides and Resources</Menu.Item>
-              <Menu.Item key="about">About</Menu.Item>
-            </Menu>
-          </Sider>
-          }
-        </>}
-        <Layout>
-          {!this.state.isMobile &&
-          <Header>
-            <div className="logo" onClick={() => this.handleNav({key: 'map'})}></div>
-            <Menu
-              theme="dark"
-              mode="horizontal"
-              style={{ lineHeight: '64px' }}
-              onClick={this.handleNav}
-              selectedKeys={[this.state.currentTab]}
-            >
-              <Menu.Item key="map">Map</Menu.Item>
-              <Menu.Item key="networks">Table View</Menu.Item>
-              <Menu.Item key="resources">Guides and Resources</Menu.Item>
-              <Menu.Item key="about">About</Menu.Item>
-            </Menu>
-          </Header>}
-          <Content style={{ padding: '0 50px' }}>
-            <div className="main-container">
-            {this.state.currentTab === 'map' && <>
-              {mapboxgl.supported() ? <>
-                <Filters
-                  setFilters={setFilters}
-                  selectedCategories={selectedCategories}
-                  absolute={true}
-                  visible={viewState ==='default'}
-                />
-                <div className={`interactive-content-${viewState}`}>
-                <MapView
-                    networks={filteredNetworks}
-                    viewState={viewState}
-                    setLatLng={setLatLng}
-                    selectedCategories={selectedCategories}
-                    resetToDefaultView={resetToDefaultView}
-                    hoveredPointId={hoveredPointId}
-                    setHoveredPoint={setHoveredPoint}
-                    bbox={masterBbox}
-                    setUsState={setUsState}
-                  />
-                  <ListView
-                    visibleCards={visibleCards}
-                    setHoveredPoint={setHoveredPoint}
-                    setFilters={setFilters}
-                    selectedCategories={selectedCategories}
-                  />
-                  </div>
-                </>: <NoWebGl />}
-              
-              <div className="tagline">Find Mutual Aid Networks and other community self-support projects near you. Reach out to these groups directly via the map above to get involved, offer resources, or submit needs requests.</div>
-              <SubmitButton
-                link='https://docs.google.com/forms/d/e/1FAIpQLScuqQtCdKsDzvTzaA2PMyVHX7xcOqbOW7N7l_0YJASV4wMBVQ/viewform'
-                description='Submit a Mutual Aid Network'
+      <Router>
+        <Layout className="layout">
```

Comment:
One thing that may make this easier to read is separating out Header and Footer either into files for functions. so `renderHeader()` and `renderFooter()` all within this file, or make Header and Footer functional components.

Reply:
@meganrm I just put up a PR addressing this: https://github.com/townhallproject/mutual-aid-networks/pull/74